src/home/actions.ts

```diff
@@ -61,6 +85,9 @@ export type ActionTypes =
   | VisitHomeAction
   | CelebratedNftFoundAction
   | NftCelebrationDisplayedAction
+  | NftRewardReadyToDisplayAction
+  | NftRewardDisplayedAction
+  | NftRewardReminderReadyToDisplayAction
```

Comment:
nit: i wonder if NftRewardReadyToDisplayAction and NftRewardReminderReadyToDisplayAction could be combined, the action + reducer + their usage seem very similar, perhaps it'd be more simple to have a single action that takes a boolean `showReminder` param?

Reply:
sure, thanks!